### DISCUSSION TITLE
macOS dropdown not working with `cmd` + `.`

### DISCUSSION THREAD

**Discussion by duwivom** [tailwindlabs/headlessui#801] [Ideas]
### What package within Headless UI are you using?

@headlessui/vue

### What version of that package are you using?

v1.0.0

### What browser are you using?

Chrome and Safari

### Reproduction repository

https://headlessui.dev/examples/vue-menu-hero

### Describe your issue

Native select closes with Escape or Command-. while this component only closes with Escape.

**Comment by RobinMalfait** [comment]
Converted this issue to a Discussion, since this is a nice idea but not really a bug. We focused on the initial implementation based on https://www.w3.org/TR/wai-aria-practices-1.2/#menubutton, and the `cmd + .` would be a nice improvement.
